gio/vendor.md

```diff
@@ -18079,7 +18079,7 @@ a `GAsyncReadyCallback`
 ## `user_data`
 user data for the callback
 <!-- trait SocketListenerExt::fn accept_finish -->
-Finishes an async accept operation. See `SocketListenerExt::accept_async`
+Finishes an async accept operation. See `SocketListener::accept_async`
```

Comment:
It not in `SocketListenerExtManual` ?

Reply:
Checking.

Reply:
It is indeed, good catch!

Reply:
Hm, this is local function and it not be in doc, so can be ignored

Reply:
I wonder if this is a bad generation coming from gir or from somewhere else...

Reply:
I fixed it anyway. :)

Reply:
But what next text unchanged is bad, gir need fix sometime
```
<!-- trait SocketListenerExt::fn accept_async -->
This is the asynchronous version of `SocketListenerExt::accept`.
```

Reply:
Damn it.

Reply:
IMHO as we have https://github.com/GuillaumeGomez/rustdoc-stripper/issues/53 too,
better update docs for this version later

Reply:
and this PR can be merged as is but not crate published

Reply:
Agreed.